cartridges/int_stickyio_sfra/cartridge/controllers/CheckoutServices.js

```diff
@@ -314,9 +323,12 @@ if (stickyioEnabled) {
                 req.session.privacyCache.set('usingMultiShipping', false);
                 usingMultiShipping = false;
             }
-
-            hooksHelper('app.customer.subscription', 'subscribeTo', [paymentForm.subscribe.checked, paymentForm.contactInfoFields.email.htmlValue], function () {});
-
+            
+            if (!sfccVersion60) {
+            	hooksHelper('app.customer.subscription', 'subscribeTo', [paymentForm.subscribe.checked, paymentForm.contactInfoFields.email.htmlValue], function () {});
+			} else {	
+            	hooksHelper('app.customer.subscription', 'subscribeTo', [paymentForm.subscribe.checked, currentBasket.customerEmail], function () {});
+			}
```

Comment:
Please correct spacing issues in lines 328-331?

Reply:
fixed

Reply:
Please correct spacing issues in line 331?